backend/zeno_backend/database/select.py

```diff
@@ -884,6 +882,23 @@ def project_state(project_uuid: str, project: Project) -> ProjectState | None:
         )
         models = [m[0] for m in model_results]
 
+        # Check whether the user or an org of the user have project edit rights
+        user_editor = db.execute_return(
+            "SELECT editor FROM user_project WHERE user_id = %s AND project_uuid = %s",
+            [user.id, project_uuid],
+        )
+        org_editor = db.execute_return(
+            "SELECT editor FROM organization_project AS r JOIN user_organization "
+            "AS o ON r.organization_id = o.organization_id "
+            "WHERE r.project_uuid = %s AND o.user_id = %s;",
+            [project_uuid, user.id],
+        )
```

Comment:
We could also do `SELECT * FROM ... WHERE r.editor = True`. Then we would not have to do the `any()`. I assume this would be cheaper and probably more clean.